lib/models/space.rb

```diff
@@ -0,0 +1,13 @@
+require 'data_mapper'
+require 'dm-postgres-adapter'
+
+class Space
+
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :name, String
+  property :description, String
```

Comment:
Description needs length greater than default of 50. Try 500 for now. `length: 500`